tools/mininet/Dockerfile

```diff
@@ -122,3 +128,4 @@ EXPOSE 50001-50100
 
 ENTRYPOINT ["mn", "--custom", "/root/stratum.py", "--switch", "stratum-bmv2", "--host", "no-offload-host", "--controller", "none"]
 
+RUN python -c "from p4.config.v1 import p4info_pb2;from google.rpc import status_pb2, code_pb2;from p4.v1 import p4runtime_pb2, p4runtime_pb2_grpc"
```

Comment:
I think it would make more sense to do this test as part of the CI script, otherwise, we create an unnecessary layer in the final image.

Reply:
This command currently fails on my machine, but it should succeed in CircleCI with this PR:
```➜  stratum git:(master) docker run --rm --entrypoint python opennetworking/mn-stratum:latest -c "from p4.config.v1 import p4info_pb2;from google.rpc import status_pb2, code_pb2;from p4.v1 import p4runtime_pb2, p4runtime_pb2_grpc"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name code_pb2```

Reply:
Sure, I just wanted something to test on my machine.
Moved to CI